contracts/strategy/rysk/RyskStrategy.sol

```diff
@@ -37,13 +39,41 @@ contract RyskStrategy is BaseStrategy {
      */
     event StrategyWithdrawalInitiated(uint256 amount);
 
+    /**
+     * Emmited when a withdrawal has been initiated.
+     *
+     * @param startingPricePerShare the price per share at the start of the
+     * yield distribution cycle.
+     * @param targetPricePerShare the price per share at the end of the yield
+     * distribution cycle.
+     * @param syncYieldStartTimestamp the timestamp of the start of the current
+     * yield distribution cycle.
+     * @param depositedAmount the sum of deposits and distributed yield in the
+     * contract.
+     */
+    event StrategySyncYield(
+        uint256 startingPricePerShare,
+        uint256 targetPricePerShare,
+        uint256 syncYieldStartTimestamp,
+        uint256 depositedAmount
+    );
+
     // rysk liquidity pool cannot be 0 address
     error RyskLiquidityPoolCannotBe0Address();
     // no withdrawal initiated
     error RyskNoWithdrawalInitiated();
     // cannot complete withdrawal in the same epoch
     error RyskCannotCompleteWithdrawalInSameEpoch();
 
+    // The length of a yield distribution cycle
+    uint256 private yieldCycleLength;
+    // The sum of deposits and distributed yield
+    uint256 private depositedAmount;
+    // The price per share at the beginning of the yield distribution cycle
+    uint256 private startingPricePerShare;
+    uint256 private targetPricePerShare;
+    uint256 private syncYieldStartTimestamp;
```

Comment:
Add missing comments here. I would suggest renaming `syncYieldStartTimestamp` to `yieldCycleStartTimestamp`